### PR TITLE
feat: Golang example

### PR DESCRIPTION
**Pull request by arianejasuwienas**
**Description**:
Golang example

### CODE REVIEW

**Review by ebadiere — DISMISSED** [review]
LGTM.

**Review by acuarica — DISMISSED** [review]
lgtm, left some optional comments

File: tools/golang-example/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.env
+hedera-golang-example-project
```
[the remark this answers is not in the document]
Comment (arianejasuwienas) [reply tools/golang-example/.gitignore:4]: Good idea, thanks. I've added this description [here](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2584/commits/08099ae397bdbee29a7f71f7c275675d31befff8#diff-0c9985b1c0aac07b6d9c7d2a939bd622f3038a211a0c2dad0653124af7005a1aR3) . Let me know if that is enough.

**Review by acuarica — APPROVED** [review]
lgtm

**Pull request merged**